browser/ui/views/split_view/split_view_location_bar_browsertest.cc

```diff
@@ -77,22 +118,30 @@ IN_PROC_BROWSER_TEST_F(SplitViewLocationBarBrowserTest,
   EXPECT_TRUE(split_view_location_bar().GetWidget()->IsVisible());
 
   // When breaking the split view
-  brave::BreakTiles(browser());
+  CloseSplitView();
 
   // Then the secondary location bar should be hidden
   base::RunLoop().RunUntilIdle();
   EXPECT_FALSE(split_view_location_bar().GetWidget()->IsVisible());
 }
 
-IN_PROC_BROWSER_TEST_F(SplitViewLocationBarBrowserTest,
+IN_PROC_BROWSER_TEST_P(SplitViewLocationBarBrowserTest,
                        URLShouldBeUpdated_WhenActiveTabChanges) {
-  brave::NewSplitViewForTab(browser());
+  OpenSplitView();
   ASSERT_EQ(1, tab_strip_model().GetIndexOfWebContents(
                    tab_strip_model().GetActiveWebContents()));
   ASSERT_EQ("about:blank",
             tab_strip_model().GetWebContentsAt(0)->GetVisibleURL().spec());
   ASSERT_EQ(u"about:blank", split_view_location_bar().url_->GetText());
 
+  // For now upstream's another page is loaded when split view is created
+  // with active tab. So, loaded newtab to use below url check.
+  if (IsSideBySideEnabled()) {
+    ASSERT_TRUE(ui_test_utils::NavigateToURLWithDisposition(
+        browser(), GURL("chrome://newtab/"), WindowOpenDisposition::CURRENT_TAB,
+        ui_test_utils::BROWSER_TEST_WAIT_FOR_LOAD_STOP));
+  }
+
```

Comment:
This code is deleted by another PR(https://github.com/brave/brave-core/pull/29124)